slurm_ops_manager/slurm_ops_base.py

```diff
@@ -446,6 +446,25 @@ def setup_nhc(self) -> bool:
 
         return status
 
+    def setup_logrotate(self):
+        """Set up logrotate profiles.
+
+        Copy logorate profiles for NHC and Slurm.
+
+        Returns True on success, False otherwise.
```

Comment:
Seems like this method never returns anything but `None`

Reply:
Copy and paste fail.